t/parent-pmc.t

```diff
@@ -14,11 +14,25 @@ use lib 't/lib';
 
 plan skip_all => ".pmc are only available with 5.6 and later" if $] < 5.006;
 
-my $no_pmc = defined &Config::non_bincompat_options
```

Comment:
`defined` returns false even if `can` returns true (the sub is predeclared in Config.pm).